units/migrate_volume.go

```diff
@@ -137,10 +138,14 @@ func (j *volumeMigrationJob) Run(ctx context.Context) {
 		}
 	}
 
-	// Set initial host to have expiration in 24 hours
-	if err := j.initialHost.SetExpirationTime(time.Now().Add(evergreen.DefaultSpawnHostExpiration)); err != nil {
-		j.AddError(errors.Wrapf(err, "setting expiration for host '%s'", j.InitialHostID))
-		return
+	// If not terminated, set initial host to have expiration in 24 hours
+	if j.initialHost.Status == evergreen.HostStopped {
```

Comment:
Why is this specific to just stopped hosts?

Reply:
If the initial host is already terminated, setting the expiration is useless afaik (we've already ensured the initial host is either stopped or terminated on L84).